lib/membrane/rtp/muxer.ex

```diff
@@ -1,5 +1,9 @@
 defmodule Membrane.RTP.Muxer do
-  @moduledoc false
+  @moduledoc """
+  Element that combines multiple streams into a single RTP stream. Each new input stream is assigned a unique SSRC, that the packets 
```

Comment:
The first line is used as a short description

```suggestion
  Element that combines multiple streams into a single RTP stream.
  
  Each new input stream is assigned a unique SSRC, that the packets 
```